sortinghat/cli/cmds/enroll.py

```diff
@@ -61,6 +64,10 @@ def enroll(ctx, uuid, organization, from_date, to_date, **extra):
 
     If the given period for that enrollment is enclosed by
     one already stored, the command will return an error.
+    This might happen when default values are already set
+    an enrollment. In the case, instead of removing an
```

Comment:
I think this should be: (...)values are already set `for`/`in` an enrollment. 
WDYT?